Falstring/azerothcore-wotlk
# update neferian head questline rev_1554113581920680700.sql

Timer updated to 2 hours from 21 seconds.

CHANGES PROPOSED:
Updated object timer to 2 hours.

ISSUES ADDRESSED:
Issue is reported in: #1702

TESTS PERFORMED:
Database. Does not produce warnings nor errors.

HOW TO TEST THE CHANGES:
use an Alliance character and then .additem 19003
start the quest from the item
follow the quest through
head should remain for 2 hours
-->

KNOWN ISSUES AND TODO LIST:
[ ]
[ ]
Target branch(es):
Master
